src/file/decrypt.rs

```diff
@@ -149,7 +149,6 @@ fn get_output_writer(out_path: PathBuf) -> Result<Box<dyn Write>, String> {
     } else {
         OpenOptions::new()
             .create_new(true)
-            .write(true)
```

Comment:
Was removal of the `.write(true)` intentional? Is it not needed because we are always creating a new file and never overwriting an existing one?

Reply:
This was from a clippy lint. We had `.write(true).append(true)`, which it flagged as redundant since it only needs append set